package.json

```diff
@@ -46,4 +46,4 @@
     "ts-jest": "^27.0.7",
     "typescript": "^4.4.4"
   }
-}
+}
```

Comment:
I think you may have an option in your editor trimming newlines at the end of files, we like those in general!

Reply:
👀 I saw this diff and I purposely committed it, thinking it was actually _adding_ the newline... my bad! 😅